h/services/subscription.py

```diff
@@ -0,0 +1,85 @@
+from typing import List
+
+from sqlalchemy import func
+from webob.cookies import SignedSerializer
+
+from h.models import Subscriptions
+from h.security import derive_key
+
+
+class InvalidUnsubscribeToken(Exception):
+    ...
+
+
+class SubscriptionService:
+    class Type:
+        REPLY = "reply"
+
+        ALL_TYPES = (REPLY,)
```

Comment:
Should this be a real Enum? Should we stop pretending this is ever going to be more than one type of subscription?

Reply:
```suggestion
        ALL = (REPLY,)
```

If this was an Enum it would be iterable